projects/packages/stats/src/class-tracking-pixel.php

```diff
@@ -105,7 +106,7 @@ public static function build_view_data() {
 					$view_data['arch_cat'] = $wp_the_query->query['category_name'] ?? $wp_the_query->query_vars['category_name'] ?? '';
 				}
 				if ( $wp_the_query->is_tag ) {
-					$view_data['arch_tag'] = $wp_the_query->query['tag'];
+					$view_data['arch_tag'] = $wp_the_query->query['tag'] ?? $wp_the_query->query_vars['tag'] ?? '';
```

Comment:
Similarly to my comment about the missing search terms above, should we continue to track cases where the tag term is not in either the `query` or `query_vars` array?

Reply:
Is this possible? If so, how common is that? In any case, if we decide to tackle that, I'd say it's a separate fix.

Reply:
From the logs shared with us, we have cases where `is_tag=true` but  the tag field in the `query` is empty. I wasn't able to reproduce it because I can't identify the root cause. All my manual tests, indicated that it could be the result of outside factors (like plugins, network).

> it's a separate fix
Just to clarify that this adjustment here is good to go.